.storybook/main.js

```diff
@@ -102,6 +105,18 @@ module.exports = {
       enforce: 'pre',
     });
 
+    config.module.rules.push({
+      test: /.+\.tsx?$/,
+      include: [modulesPath],
+      exclude: /examples|stories|spec|codemod|docs/,
+      loaders: [
+        {
+          loader: path.resolve(__dirname, 'symbol-doc-loader'),
+        },
+      ],
+      enforce: 'pre',
+    });
+
```

Comment:
This is the entry point for Webpack/Storybook. The `symbol-doc-loader` is a Webpack loader that updates the symbol doc array with all docs found in each source file.